src/main/java/com/ea/restaurant/entities/AppClientScope.java

```diff
@@ -0,0 +1,24 @@
+package com.ea.restaurant.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+@Entity
+@Table(name = "app_clients_scopes")
+public class AppClientScope extends BaseEntity {
+  @NotNull private Integer appClientId;
```

Comment:
Isn't this `Long`?

Reply:
i Think is not necessary for the moment as the amount of clients that can login in my service is not to big. i am following the types i used in the BD, for this case in the BD i used Int32.

Reply:
But mostly all ids are Long and the relation between may affect in the future. I would suggesto to keep it consistent unless this is a problem for you.

Reply:
Done. I changed the id type to Long.  Will be pending to create a migration to change the type in the BD.